packages/webservice/app/src/test/kotlin/com/yahoo/navi/ws/test/integration/ReportTest.kt

```diff
@@ -99,6 +99,7 @@ class ReportTest : IntegrationTest() {
             |"filters":[{"field":"id", "values":["-1", "102", "103"], "type":"dimension", "parameters":{}, "operator":"include"}], 
             |"requestVersion":"2.0", 
             |"sorts":[], 
+            |"rollup": {"columns":[], "grandTotal":false},
```

Comment:
Can we name this `columnsCids` so it is clear what is expected? Also, we could support canonical names or column objects in the future (if need be).
```suggestion
            |"rollup": {"columnCids":[], "grandTotal":false},
```

Reply:
so we talked offline, and we decided to punt this for now, as it would be a much lower LOE to change this once the feature is complete and merged;